render/json/default.py

```diff
@@ -223,7 +223,9 @@ def list(self, skellist, action="list", params=None, **kwargs):
 		if skellist:
 			for skel in skellist:
 				skels.append(self.renderSkelValues(skel))
-			res["structure"] = self.renderSkelStructure(skellist.baseSkel)
+
+			if skellist.baseSkel:
+				res["structure"] = self.renderSkelStructure(skellist.baseSkel)
```

Comment:
This is so ugly, I hate it... in any way, there must be an else case setting it to None...